@jcayouette Your changes do not work. My tests fail now that have been working before.

```bash
./uyuni-docs-helper -p uyuni -c antora-uyuni-en -j nodeg -t fix_container
[INFO] Pulling the latest container image ghcr.io/nodeg/uyuni-docs-helper:fix_container...
Trying to pull ghcr.io/nodeg/uyuni-docs-helper:fix_container...
(...)
cd /home/docs/uyuni-docs/
/home/docs/uyuni-docs//use_po.sh
UTF-8 "\xE7" does not map to Unicode at /usr/lib/perl5/vendor_perl/5.42.0/Locale/Po4a/Po.pm line 353.
cd /home/docs/uyuni-docs/
sed -E  -e 's/\*\*\*\*\*\ xref\:(.*)\.adoc\[(.*)\]/include\:\:modules\/installation-and-upgrade\/pages\/\1\.adoc\[leveloffset\=\+4\]/' -e 's/\*\*\*\*\ xref\:(.*)\.adoc\[(.*)\]/include\:\:modules\/installation-and-upgrade\/pages\/\1\.adoc\[leveloffset\=\+3\]/' -e 's/\*\*\*\ xref\:(.*)\.adoc\[(.*)\]/include\:\:modules\/installation-and-upgrade\/pages\/\1\.adoc\[leveloffset\=\+2\]/' -e 's/\*\*\ xref\:(.*)\.adoc\[(.*)\]/include\:\:modules\/installation-and-upgrade\/pages\/\1\.adoc\[leveloffset\=\+1\]/' -e 's/\*\ xref\:(.*)\.adoc\[(.*)\]/include\:\:modules\/installation-and-upgrade\/pages\/\1\.adoc\[leveloffset\=\+0\]/' -e 's/\*\*\*\*\ (.*)/==== \1/' -e 's/\*\*\*\ (.*)/=== \1/' -e 's/\*\*\ (.*)/== \1/' -e 's/\*\ (.*)/= \1/' translations/en/modules/installation-and-upgrade/nav-installation-and-upgrade-guide.adoc > translations/en/modules/installation-and-upgrade/nav-installation-and-upgrade-guide.pdf.en.adoc
cd /home/docs/uyuni-docs/
cd /home/docs/uyuni-docs//translations/en && LANG=en_US.utf8 LC_ALL=en_US.utf8 LC_TYPE=en_US.utf8 asciidoctor-pdf -r /home/docs/uyuni-docs//extensions/xref-converter.rb -a lang=en -a pdf-themesdir=branding/pdf/themes/ -a pdf-theme=uyuni -a pdf-fontsdir=branding/pdf/fonts -a productname='Uyuni' -a uyuni-content=true -a examplesdir=modules/installation-and-upgrade/examples -a imagesdir=modules/installation-and-upgrade/assets/images -a revdate="February 18 2026"  --base-dir . --out-file /home/docs/uyuni-docs//build/en/pdf/uyuni_installation-and-upgrade_guide.pdf modules/installation-and-upgrade/nav-installation-and-upgrade-guide.pdf.en.adoc --trace
bash: line 1: asciidoctor-pdf: command not found
make: *** [Makefile.section.functions:21: pdf-installation-and-upgrade-uyuni-en] Error 127
[ERROR] There were errors! Please review the log!
```

I ran the new container built with your changes, and the gems are not working as expected.
```bash
docs@bccc873f6db6:~> which asciidoctor
docs@bccc873f6db6:~> asciidoctor
bash: asciidoctor: command not found
docs@bccc873f6db6:~> asciidoctor-pdf
bash: asciidoctor-pdf: command not found
docs@bccc873f6db6:~>
```

--- a/image/Dockerfile
+++ b/image/Dockerfile
@@ -8,8 +8,8 @@ ENV GITREF=master
 ENV PRODUCT=uyuni
 ENV COMMAND=all-uyuni
 ENV SERVE=0
-ENV GEM_HOME="${REGULARUSER}/gems"
-ENV PATH=$GEM_HOME/bin:$GEM_HOME/gems/bin:$PATH
+ENV GEM_HOME="/home/${REGULARUSER}/gems"
+ENV PATH=$GEM_HOME/bin:$PATH
 
 USER root
 
@@ -27,10 +27,10 @@ RUN zypper -q -n refresh && \
     zypper -q -n install -t pattern devel_C_C++ && \
     zypper -q -n install \
     # Base requirements
-    glibc-locale glibc-i18ndata git libyaml-devel readline-devel sqlite3-devel \
+    glibc-locale-base glibc-locale glibc-i18ndata git libyaml-devel readline-devel sqlite3-devel zip \
     # Python 3 and libraries
     python313-devel python313-Jinja2 python313-PyYAML \
-    # Pearl
+    # Perl
     perl \
     # Ruby 3 for building PDFs
     libopenssl-devel ruby3.4 ruby3.4-devel ruby3.4-rubygem-bundler \
@@ -53,7 +53,7 @@ RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.3/install.sh | b
 RUN source ~/.bashrc && \
     npm install -g npm@latest \
     @antora/cli@3.1 \
-    @antora/site-generator@3.1\
+    @antora/site-generator@3.1 \
     @antora/lunr-extension \
     @asciidoctor/tabs
 
@@ -69,7 +69,8 @@ RUN localedef -i ja_JP -f UTF-8 ja_JP.UTF-8 && \
     localedef -i zh_CN -f UTF-8 zh_CN.UTF-8 && \
     localedef -i ko_KR -f UTF-8 ko_KR.UTF-8 && \
     localedef -i en_US -f UTF-8 en_US.UTF-8 && \
-    locale -a | grep -E "ja_JP|zh_CN|ko_KR|en_US"
+    localedef -i en_US -f UTF-8 en && \
+    locale -a | grep -E "ja_JP|zh_CN|ko_KR|en_US|en"
 
 COPY startup.sh /usr/local/bin/startup.sh
 
@@ -86,5 +87,5 @@ ENV LANGUAGE=en_US
 ENV LC_ALL=en_US.UTF-8
 
 EXPOSE 8000
-WORKDIR /home/docs
+WORKDIR /home/${REGULARUSER}
 CMD ["/usr/local/bin/startup.sh"]

